src/components/Table/__snapshots__/Table.story.storyshot

```diff
@@ -47251,10 +47442,11 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Table
                 </span>
               </td>
               <td
-                className="RowActionsCell__StyledTableCell-sc-1toqqj7-0 bGkBkT"
+                className="iot--row-actions-cell--table-cell"
               >
                 <div
-                  className="iot--row-actions-container RowActionsCell__RowActionsContainer-sc-1toqqj7-1 jpHMyF"
+                  className="iot--row-actions-container"
+                  isRowExpanded={false}
```

Comment:
Make sure this prop isn't getting attached. I'm not sure if I put a comment on the script file